polynote-frontend/polynote/ui/component/notebook/notebook.ts

```diff
@@ -162,6 +164,18 @@ export class Notebook extends Disposable {
             }
         }).disposeWith(this)
 
+        this.notebookState.loaded.then(() => {
+            this.resizeObserver = Main.get.splitView.onResize(width => {
+                if (ServerStateHandler.state.currentNotebook === path) {
+                    const cellWidth = width - 66;
```

Comment:
where does 66 come from?

Reply:
It's the width of a cell editor in comparison to the overall center area. 😬

Reply:
Gotcha. If we can't derive it from CSS somehow that's fine but maybe a comment to that effect would be useful for the future.

Reply:
I guess it could be derived once from CSS.

Reply:
Changed this to measure only first code cell rather than hardcoding this math (seems to work out just as well)